dist/tools/hydracomment/hydracomment.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2023 Koen Zandberg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+# @author   Koen Zandberg <koen@bergzand.net>
+
+"""
+Place identical comments on multiple PRs.
+Note that with great power comes great responsibility
+"""
+
+import os
+import sys
+import argparse
+from agithub.GitHub import GitHub
+
+ORG = "RIOT-OS"
+REPO = "RIOT"
+GITHUBTOKEN_FILE = ".riotgithubtoken"
+
+
+def main():
+    keyfile = os.path.join(os.environ["HOME"], GITHUBTOKEN_FILE)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-k",
+        "--keyfile",
+        type=argparse.FileType("r"),
+        default=keyfile,
+        help="File containing github token",
+    )
+    parser.add_argument(
+        "-c",
+        "--comment",
+        default="Bors merge",
```

Comment:
```suggestion
        default="bors merge",
```

I'm not sure if it is case sensitive, but so far we always used the lower case version.